tests/sniffer.py

```diff
@@ -7,36 +7,89 @@
 
 import os
 import subprocess
-
+import json
 from opts import debug, err, opts, status
+from time import time
 
 
 class Sniffer:
     '''Captures packets on an interface.'''
+
     def __init__(self, interface: str):
         self.interface = interface
         self.tcpdump_proc = None
+        self.outputfiles = {}
```

Comment:
There is no need for this dictionary. At any point, there is only one running capture, thus only one running outputfile. All the get_packet_capture() calls will be for that last outputfile.

Note that multiple captures in parallel is not possible because they're all capturing the same packets.